titiler/application/setup.py

```diff
@@ -6,6 +6,7 @@
     long_description = f.read()
 
 inst_reqs = [
+    "rio-cogeo>=2.2,<2.3",
```

Comment:
I don't think we need `,<2.3` here, we use few rio-cogeo function and I don't see how it will be breaking in 2.3